UIKit/Tasks/ThanksViewController.swift

```diff
@@ -5,9 +5,79 @@ import UIKit
 
 // Класс успешного заказа
 class ThanksViewController: UIViewController {
+    // MARK: - Constants
+
+    private let cancelImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "x")
+        image.frame = CGRect(x: 20, y: 60, width: 14, height: 14)
+        return image
+    }()
+
+    private let shareLabel: UILabel = {
+        let label = UILabel()
+        label.text = """
+        Разскажи о насъ другу, отправь ему промокодъ на безплатный напитокъ и получи скидку 10% на слѣдующій заказъ.
+        """
+
+        label.font = UIFont(name: "Verdana", size: 16)
+        label.textColor = #colorLiteral(red: 0.5294118524, green: 0.5294118524, blue: 0.5294118524, alpha: 1)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.frame = CGRect(x: 35, y: 440, width: 315, height: 89)
+        return label
+    }()
+
+    private let thxImage: UIImageView = {
```

Comment:
```suggestion
    private let (пишешь что за изображение или за что отвечает)Image: UIImageView = {. ну это как я понял
```
за такой нейминг могут заставить отжиматься)